packages/core/src/types.ts

```diff
@@ -1,58 +1,16 @@
 import * as CSS from 'csstype';
+import { UNSUPPORTED_CSS_PROPERTIES } from './constants';
 
 export type GriffelStylesCSSValue = string | 0;
 
-type GriffelStylesUnsupportedCSSProperties = {
-  // We don't support expansion of CSS shorthands
-  animation?: never;
-  background?: never;
-  border?: never;
-  borderBlock?: never;
-  borderBlockEnd?: never;
-  borderBlockStart?: never;
-  borderBottom?: never;
-  borderColor?: never;
-  borderImage?: never;
-  borderInline?: never;
-  borderInlineEnd?: never;
-  borderInlineStart?: never;
-  borderLeft?: never;
-  borderRadius?: never;
-  borderRight?: never;
-  borderStyle?: never;
-  borderTop?: never;
-  borderWidth?: never;
-  columnRule?: never;
-  flex?: never;
-  flexFlow?: never;
-  font?: never;
-  gap?: never;
-  grid?: never;
-  gridArea?: never;
-  gridColumn?: never;
-  gridGap?: never;
-  gridRow?: never;
-  gridTemplate?: never;
-  listStyle?: never;
-  margin?: never;
-  mask?: never;
-  maskBorder?: never;
-  offset?: never;
-  outline?: never;
-  overflow?: never;
-  padding?: never;
-  placeItems?: never;
-  placeSelf?: never;
-  textDecoration?: never;
-  textEmphasis?: never;
-  transition?: never;
-};
+export type GriffelStylesUnsupportedCSSProperties = Record<keyof typeof UNSUPPORTED_CSS_PROPERTIES, never>;
```

Comment:
As we are now `csstype@3` this could be improved:

```ts
type GriffelStylesUnsupportedCSSProperties = Record<
  keyof CSS.StandardShorthandProperties | keyof CSS.VendorShorthandProperties,
  never
>;
```

We should use this instead of our types.

Reply:
well it depends if we want to enable css property stripping in all environments... because in that case we should probably follow another source of truth... I'll try it out and see how it goes tho

Reply:
I can type `UNSUPPORTED_CSS_PROPERTIES` with that type, the only exception is `gridGap` which is in `CSS.ObsoleteProperties`, should I include that in the type union ?

Reply:
> I can type `UNSUPPORTED_CSS_PROPERTIES` with that type, the only exception is `gridGap` which is in `CSS.ObsoleteProperties`, should I include that in the type union ?

Yes, please. We can include `gridGap` manually

Reply:
This also resulted in extra vendor shorthands... do we want them ?

Reply:
> This also resulted in extra vendor shorthands... do we want them ?

Let's skip for now (until someone complains) 😅

Reply:
OK I removed all the vendor shorthands